src/components/date-picker/date-picker.js

```diff
@@ -297,6 +298,9 @@ class DatePicker extends mixin(createComponent, initComponentBySearch, handles)
   _formatDate = date => this.calendar.formatDate(date, this.calendar.config.dateFormat);
 
   release() {
+    if (this._rangeInput) {
+      this._rangeInput.remove();
```

Comment:
`.remove()` is not supported by IE11 - We can do e.g.:

```javascript
if (this._rangeInput && this._rangeInput.parentNode) {
  this._rangeInput.parentNode.removeChild(this._rangeInput);
}
```

Reply:
Thanks, I adjusted accordingly.

I tested in Chrome and it still works - I'm on a mac so I have no good way to verify in IE.